coalib/bears/Bear.py

```diff
@@ -26,6 +26,14 @@
 
 class Debugger(pdb.Pdb):
 
+    def __init__(self, *args, bear=None, **kwargs):
+        if bear is None:
+            raise ValueError('Invalid argument passed. Bear object '
+                             'must passed as a argument for inspecting bear '
+                             'settings in debugging environment.')
```

Comment:
I would actually just say "Bear missing". That is short and easy to understand :P